### ISSUE TITLE
ConTeXt can't find mtx-context.lua unless TEXMF is manually set

### ISSUE THREAD

**Issue by anderspapitto** [NixOS/nixpkgs#19279]
``` bash
[anders@gurney:~/playground]
$ git clone git@github.com:mszep/pandoc_resume
Cloning into 'pandoc_resume'...
remote: Counting objects: 153, done.        
remote: Total 153 (delta 0), reused 0 (delta 0), pack-reused 153        
Receiving objects: 100% (153/153), 58.20 KiB | 0 bytes/s, done.
Resolving deltas: 100% (64/64), done.
[anders@gurney:~/playground]
$ nix-shell -p haskellPackages.pandoc -p 'texlive.combine { inherit (texlive) scheme-context; }'

[nix-shell:~/playground]$ cd pandoc_resume/

[nix-shell:~/playground/pandoc_resume]$ make pdf
pandoc --standalone --template style_chmduquesne.tex \
--from markdown --to context \
-V papersize=A4 \
-o resume.tex resume.md; \
context resume.tex

mtxrun          | unknown script 'context.lua' or 'mtx-context.lua'
make: *** [Makefile:5: resume.pdf] Error 1

[nix-shell:~/playground/pandoc_resume]$ nativeBuildInputsAsBashArray=($nativeBuildInputs)

[nix-shell:~/playground/pandoc_resume]$ TEXMF=${nativeBuildInputsAsBashArray[1]}/share/texmf make pdf
# output elided, but it works just fine
```

I shouldn't have to set TEXMF myself

I'm on current NixOS unstable

**Comment by AndersonTorres** [comment]
It was solved?

**Comment by anderspapitto** [comment]
no idea but I haven't been using this, so I'll close it

**Comment by AndersonTorres** [comment]
I have tested it and the workaround is still needed. Maybe it can be useful to the package maintainers.

**Comment by xeji** [comment]
fixed in #47426
